sql/common/src/test/scala/org/apache/sedona/sql/functionTestScala.scala

```diff
@@ -1921,4 +1921,18 @@ class functionTestScala extends TestBaseScala with Matchers with GeometrySample
       assertEquals(expected, actual)
     }
   }
+
+  it("should pass ST_Force3D") {
+    val geomTestCases = Map(
+      ("'LINESTRING (0 1, 1 0, 2 0)'") -> "'LINESTRING Z(0 1 1, 1 0 1, 2 0 1)'",
+      ("'LINESTRING Z(0 1 3, 1 0 3, 2 0 3)'") -> "'LINESTRING Z(0 1 3, 1 0 3, 2 0 3)'",
+      ("'LINESTRING EMPTY'") -> "'LINESTRING EMPTY'"
+    )
+    for (((geom), expectedResult) <- geomTestCases) {
+      val df = sparkSession.sql(s"SELECT ST_AsText(ST_Force3D(ST_GeomFromWKT($geom), 1)) AS geom, " + s"$expectedResult")
```

Comment:
Please add the test case that has no default zValue supplied

Reply:
Added

Reply:
Same here: add a test case like ST_Force3D(geom), no z value is supplied. This will test the correctness of the the optional parameter.